packages/babel-plugin-fbtee/src/babel-processors/FbtFunctionCallProcessor.tsx

```diff
@@ -666,9 +657,10 @@ export default class FbtFunctionCallProcessor {
     const fbtRuntimeArgs = [];
     for (const child of fbtNode.children) {
       if (
-        child instanceof FbtParamNode &&
-        child.options.gender == null &&
-        child.options.number == null
+        child.type === 'list' ||
+        (child.type === 'param' &&
```

Comment:
Yup, that's a good point. I generally try to avoid instanceof as much as possible in JavaScript because you might end up with a copy of the same class and spend hours debugging it. Of course, that shouldn't happen these days or with this project, but it's something that can trip you up and TypeScript can refine the types this way too.